src/doc/doc-builders.js

```diff
@@ -41,6 +65,11 @@ function align(n, contents) {
   return { type: "align", contents, n };
 }
 
+/**
+ * @param {Doc} contents
+ * @param {Object} [opts] - TBD ???
```

Comment:
https://github.com/prettier/prettier/blob/master/commands.md#group

Reply:
What about `opts.id` below?

Any pointers to what type we should expect for `opts.id`?

Should we consider adding it to GroupOpts?

Reply:
> https://github.com/prettier/prettier/blob/master/commands.md#group

Done.

My questions about opts.id still stand.

Reply:
> My questions about opts.id still stand.

Answered in <https://github.com/prettier/prettier/pull/6770#discussion_r341402982>, resolving this now.